DTS-STN/passport-status
# Add next translate

## [ADO-XXX](https://dev.azure.com/DTS-STN/passport-status/_workitems/edit/xxx)

### Description

List of proposed changes:

- Add next-translate library is to keep the translations as simple as possible in a Next.js environment.

### What to test for/How to test

1. Run `npm install` to install the new package
2. Run the application in `dev` or `production`
3. Browse the application and toogle the language
4. Fields error message should be translated as well

### Additional Notes

next-translate information and guide
https://github.com/aralroca/next-translate
